models/Psychologists.js

```diff
@@ -0,0 +1,85 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const availabilityDaySchema = new Schema({
+  availability: {
```

Comment:
if a user sets a day as `availability: false`, should it be set to `from` and `to` anyway?

Reply:
@estefrare 

Perhaps is not necessary, but anyway always having those might be useful as a default value or some kind of retentive memory of the last values configured on that day.  So, i believe the best option is not to require `from` and `to`, but assign them default values.

Also, I have been reading about [mongoose conditional validators](https://mongoosejs.com/docs/validation.html#built-in-validators), a pretty cool option there. Anyway, it helps out when creating a new document, but the conditional validation is not executed when we're updating a document, or I couldn't find how to execute the validation on update.